samples/dubbogo/simple/triple/docker/docker-health-check.sh

```diff
@@ -15,6 +15,5 @@
 #  limitations under the License.
 #
 
-sleep 5
+sleep 30
 curl http://127.0.0.1:8848/nacos/v1/console/health/liveness
-sleep 10
```

Comment:
make curl call to trigger nacos lazy initialization and sleep for waiting completely initialization. it's better to keep it unchaned.